.circleci/ci-oldest-reqs.txt

```diff
@@ -0,0 +1,5 @@
+cloudpickle==1.1.1
+deprecation==2
+jinja2==2.10
+signac==1.0.0
```

Comment:
I'd like to delete the pinning for `signac` from the "oldest" requirements and let that be handled by the `${SIGNAC_VERSION}` variable.
```suggestion
```